source/vocab/agents.ttl

```diff
@@ -75,6 +75,48 @@
     rdfs:subClassOf :BibliographicAgent;
     owl:equivalentClass bf2:Meeting, bibo:Conference .
 
+
+# Agent membership properties
+
+:hasMember a owl:ObjectProperty ;
+    rdfs:label "har medlem"@sv , "has member"@en ;
+    sdo:domainIncludes :Organization, :Family ;
+    sdo:rangeIncludes :Agent ;
+    owl:equivalentProperty sdo:member ;
```

Comment:
sdo seems not to include family when talking about member so equivalent is maybe not right.

Reply:
Maybe not; but they're just "*includes", so it's not meaningful beyond structural hints (which aren't entailed).